server/client_rest.go

```diff
@@ -70,6 +71,27 @@ func (client *clientRest) NodeSendStats(nodeKey string, sessionList []dto.Sessio
 	return nil
 }
 
+func (client *clientRest) Proposals(nodeKey string) (proposals []dto_discovery.ServiceProposal, err error) {
+	response, err := client.doRequest("GET", "proposals", dto.ProposalsRequest{
```

Comment:
Shouldn't it be like "/proposals?byNodeKey=<ourpassedkey>" ? Get cannot have body

Reply:
Yes, will fix.

Reply:
Fixed.